connectors/airwatch/pom.xml

```diff
@@ -160,6 +161,87 @@
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>io.fabric8</groupId>
+                        <artifactId>docker-maven-plugin</artifactId>
+
+                        <configuration>
+                            <images>
+                                <image>
+                                    <name>airwatch-connector</name>
+                                    <alias>airwatch-connector</alias>
+                                    <build>
+                                        <assembly>
+                                            <inline>
+                                                <files>
+                                                    <file>
+                                                        <source>${project.basedir}/src/main/resources/application.yml</source>
+                                                        <outputDirectory>./</outputDirectory>
+                                                    </file>
+                                                    <file>
+                                                        <source>${project.basedir}/src/test/resources/docker/managed-apps.yml</source>
+                                                        <outputDirectory>./</outputDirectory>
+                                                    </file>
+                                                    <file>
+                                                        <source>${project.basedir}/src/test/resources/docker/application.properties</source>
+                                                        <outputDirectory>./</outputDirectory>
+                                                    </file>
+
+                                                </files>
+                                            </inline>
+                                            <targetDir>/opt/vmware/airwatch-connector</targetDir>
+                                        </assembly>
+                                        <from>openjdk:8u151-jre</from>
+                                        <workdir>/opt/vmware/airwatch-connector</workdir>
+                                        <entryPoint>
+                                            <shell>
+                                                java -server -XX:+UnlockExperimentalVMOptions \
+                                                 -Xss256k \
+                                                 -XX:+UseCGroupMemoryLimitForHeap \
+                                                 -XX:MaxRAMFraction=1 \
+                                                 -XX:+HeapDumpOnOutOfMemoryError \
+                                                 -XX:HeapDumpPath=/tmp \
+                                                 -jar /opt/vmware/airwatch-connector/airwatch-connector-${project.version}.jar \
+                                                 --logging.level.com.vmware=${container.logging.level} \
```

Comment:
Take this one out too.  Let the user define the logging.level of things.

Reply:
yes.. not required.. i have removed it.